avocado/utils/podman.py

```diff
@@ -129,12 +129,16 @@ async def get_container_info(self, container_id):
         :rtype: dict
         """
         try:
-            _, stdout, _ = await self.execute("ps", "--all", "--format=json")
+            _, stdout, _ = await self.execute(
+                "ps", "--all", "--format=json", "--filter", f"id={container_id}"
+            )
         except PodmanException as ex:
             raise PodmanException(
                 f"Failed getting information about container:" f" {container_id}."
             ) from ex
         containers = json.loads(stdout.decode())
+        if len(containers) == 1:
```

Comment:
IMO the following for loop will do the same thing although that there is only one container. So I don't understand this if statement. IMO it is not necessary optimization.